tests/test_repository.py

```diff
@@ -1770,3 +1770,53 @@ def test_get_touched_functions():
     touched_functions = repository.get_touched_functions(metrics["spaces"], [], [4],)
 
     assert touched_functions == {("outer_func", 1, 6), ("inner_func", 3, 5)}
+
+
+def test_get_commits():
+    # By default get_commits utilizes the following parameters:
+    #   include_no_bug: bool = False
+    #   include_backouts: bool = False
+    #   include_ignored: bool = False
+
+    # 10 mock commits, 1 ignored, 1 backouts, 1 no_bug, 1 no_bug and ignored, 1 no_bug, ignored, and backouts
+    retrieved_commits = list(repository.get_commits())
+    assert len(retrieved_commits) == 5
+    assert retrieved_commits[0]["node"] == "9d576871fd33bed006dcdccfba880a4ed591f870"
+
+    retrieved_commits = list(repository.get_commits(include_backouts=True))
+    assert len(retrieved_commits) == 6
+    assert retrieved_commits[5]["node"] == "ec01c146f756b74d18e4892b4fd3aecba00da93e"
```

Comment:
This (and similar) assertions might start failing if we change the order of the commits in the mock file, you could do:
```
assert "ec01c146f756b74d18e4892b4fd3aecba00da93e" in {c["node"] for c in retrieved_commits}
```